src/components/bodyImageHalfWidth.tsx

```diff
@@ -0,0 +1,52 @@
+// ----- Imports ----- //
+
+import React, { FC } from 'react';
+import { css, SerializedStyles } from '@emotion/core';
+import Img from 'components/img';
+import { BodyImageProps as Props } from 'image';
+import { remSpace } from '@guardian/src-foundations';
+
+// ----- Setup ----- //
+
+const size = `calc(50% - ${remSpace[1]})`;
+
+// ----- Component ----- //
+
+const styles = css`
+    margin: ${remSpace[1]} 0 0 0;
+    display: inline-block;
+    width: ${size};
+
+    figcaption {
+        display: none;
+    }
```

Comment:
Does this mean that you can pass a `figcaption`, it will be rendered in the DOM, but will be hidden by CSS? Would it be preferable not to accept `children` for half width images?